native/ilaenv.go

```diff
@@ -0,0 +1,375 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+// Ilaenv returns algorithm tuning parameters for the algorithm given by the
+// input string. ispec specifies the parameter to return.
+//  1: The optimal block size
+//  2: The minimum block size for which the algorithm should be used.
+//  3: The crossover point below which an unblocked routine should be used.
+//  4: The number of shifts.
+//  5: The minumum column dimension for blocking to be used.
+//  6: The crossover point for SVD (to use QR factorization or not).
+//  7: The number of processors.
+//  8: The crossover point for multishift in QR and QZ methods for nonsymmetric eigenvalue problems.
+//  9: Maximum size of the subproblems in divide-and-conquer algorithms.
+//  10: ieee NaN arithmetic can be trusted not to trap.
+//  11: infinity arithmetic can be trusted not to trap.
+func (Implementation) Ilaenv(ispec int, s string, opts string, n1, n2, n3, n4 int) int {
+	// TODO(btracey): Relpace this with a constant lookup? A list of constants?
```

Comment:
I was thinking about this. Ilaenv is horrible, but do we need to keep it to allow compatibility with the cgo code?